Handle xml elements that onto-viewer does not recognise

We should be able top handle xml elements that onto-viewer does not recognise without terminating the process.




An example of the issue is below:



```
2022-10-31 11:19:26.131 INFO 198 --- [ main] o.e.s.o.t.OntoViewerToolkitApplication : Started OntoViewerToolkitApplication in 1.969 seconds (JVM running for 2.499) 2022-10-31 11:19:26.133 DEBUG 198 --- [ main] o.s.b.a.ApplicationAvailabilityBean : Application availability state LivenessState changed to CORRECT 2022-10-31 11:19:26.138 DEBUG 198 --- [ main] o.e.s.o.t.OntoViewerToolkitCommandLine : Current working directory: /var/lib/jenkins/workspace/iof_master@2 2022-10-31 11:19:26.139 DEBUG 198 --- [ main] o.e.s.o.t.OntoViewerToolkitCommandLine : Property source 'systemProperties' content: 'awt.toolkit' = 'sun.awt.X11.XToolkit', 'java.specification.version' = '11', 'sun.cpu.isalist' = '', 'sun.jnu.encoding' = 'UTF-8', 'java.class.path' = '/usr/share/java/onto-viewer/onto-viewer-toolkit.jar', 'java.vm.vendor' = 'Alpine', 'sun.arch.data.model' = '64', 'logging.level.org.edmcouncil.spec.ontoviewer.toolkit' = 'DEBUG', 'java.vendor.url' = 'https://alpinelinux.org/', 'user.timezone' = 'GMT', 'os.name' = 'Linux', 'java.vm.specification.version' = '11', 'sun.java.launcher' = 'SUN_STANDARD', 'user.country' = 'US', 'sun.boot.library.path' = '/usr/lib/jvm/java-11-openjdk/lib', 'sun.java.command' = '/usr/share/java/onto-viewer/onto-viewer-toolkit.jar --data /input/iof/AboutIOFProd.rdf --output /var/tmp/console.txt --ontology-mapping /input/iof/catalog-v001.xml --goal consistency-check', 'jdk.debug' = 'release', 'sun.cpu.endian' = 'little', 'user.home' = '?', 'user.language' = 'en', 'java.specification.vendor' = 'Oracle Corporation', 'java.version.date' = '2022-08-12', 'java.home' = '/usr/lib/jvm/java-11-openjdk', 'file.separator' = '/', 'java.vm.compressedOopsMode' = 'Zero based', 'line.separator' = ' ', 'java.specification.name' = 'Java Platform API Specification', 'java.vm.specification.vendor' = 'Oracle Corporation', 'FILE_LOG_CHARSET' = 'UTF-8', 'java.awt.graphicsenv' = 'sun.awt.X11GraphicsEnvironment', 'java.awt.headless' = 'true', 'logging.level.org.edmcouncil.spec.ontoviewer' = 'INFO', 'java.protocol.handler.pkgs' = 'org.springframework.boot.loader', 'sun.management.compiler' = 'HotSpot 64-Bit Tiered Compilers', 'java.runtime.version' = '11.0.16.1+1-alpine-r0', 'user.name' = '?', 'path.separator' = ':', 'os.version' = '5.4.17-2136.312.3.4.el8uek.x86_64', 'java.runtime.name' = 'OpenJDK Runtime Environment', 'logging.level.org.springframework' = 'INFO', 'file.encoding' = 'UTF-8', 'spring.beaninfo.ignore' = 'true', 'java.vm.name' = 'OpenJDK 64-Bit Server VM', 'java.vendor.url.bug' = 'https://gitlab.alpinelinux.org/alpine/aports/issues', 'java.io.tmpdir' = '/tmp', 'java.version' = '11.0.16.1', 'user.dir' = '/var/lib/jenkins/workspace/iof_master@2', 'os.arch' = 'amd64', 'java.vm.specification.name' = 'Java Virtual Machine Specification', 'PID' = '198', 'java.awt.printerjob' = 'sun.print.PSPrinterJob', 'sun.os.patch.level' = 'unknown', 'CONSOLE_LOG_CHARSET' = 'UTF-8', 'java.library.path' = '/usr/lib/jvm/java-11-openjdk/lib/server:/usr/lib/jvm/java-11-openjdk/lib:/usr/lib/jvm/java-11-openjdk/../lib:/usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib', 'java.vm.info' = 'mixed mode', 'java.vendor' = 'Alpine', 'java.vm.version' = '11.0.16.1+1-alpine-r0', 'sun.io.unicode.encoding' = 'UnicodeLittle', 'java.class.version' = '55.0', 2022-10-31 11:19:26.140 DEBUG 198 --- [ main] o.e.s.o.t.OntoViewerToolkitCommandLine : Property source 'systemEnvironment' content: 'PATH' = '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/lib/jvm/java-11-openjdk/bin:/usr/share/java/jena/latest/bin', 'RUN_DISPLAY_URL' = 'https://jenkins.edmcouncil.org/job/iof/job/master/12/display/redirect', 'family_product_branch_tag' = 'iof/ontology/master/latest', 'HUDSON_HOME' = '/var/lib/jenkins', 'RUN_CHANGES_DISPLAY_URL' = 'https://jenkins.edmcouncil.org/job/iof/job/master/12/display/redirect?page=changes', 'JENKINS_NODE_COOKIE' = 'cef2d96f-8018-4cb5-b346-a5b09d20b01d', 'LD_LIBRARY_PATH' = '/usr/lib/jvm/java-11-openjdk/lib/server:/usr/lib/jvm/java-11-openjdk/lib:/usr/lib/jvm/java-11-openjdk/../lib', 'JENKINS_SERVER_COOKIE' = 'durable-abb031611ebe52c75c76dea2be0d9b4475ddab7c9617c9009b8a2e93582932f1', 'LS_OPTIONS' = '--color=auto', 'PWD' = '/var/lib/jenkins/workspace/iof_master@2', 'JENKINS_URL' = 'https://jenkins.edmcouncil.org/', 'tag_root' = '/output/iof/ontology/master/latest', 'LANGUAGE' = 'en_US.UTF-8', 'STAGE_NAME' = 'Hygiene Tests', 'JENA_RIOT' = '/usr/share/java/jena/latest/bin/riot', 'PAGER' = 'less', 'BRANCH_IS_PRIMARY' = 'true', 'JENKINS_HOME' = '/var/lib/jenkins', 'RJPP_LOCAL_MARKER' = 'AboutIOFDev.rdf', 'NODE_NAME' = 'idmp-docker', 'SED' = 'sed', 'BUILD_DISPLAY_NAME' = '#12', 'LC_ALL' = 'en_US.UTF-8', 'HYGIENE_TEST_PARAMETER_VALUE' = 'industrialontologies', 'SCRIPT_DIR' = '/publisher', 'spec_root_url' = 'https://purl.industrialontologies.org', 'SHLVL' = '3', 'WORKSPACE_TMP' = '/var/lib/jenkins/workspace/iof_master@2@tmp', 'spec_family_root' = '/output/iof', 'PYTHON3' = 'python3', 'JQ' = 'jq', 'product_root' = '/output/iof/ontology', 'TERM' = 'xterm', 'GIT_AUTHOR' = 'mereolog', 'RJPP_SCM_URL' = 'https://github.com/edmcouncil/devops', 'JENA_ARQ' = '/usr/share/java/jena/latest/bin/arq', 'ontology_product_tag_root' = '/output/iof/ontology/master/latest', 'HYGIENE_TEST_PARAMETER' = '<HYGIENE_TESTS_FILTER_PARAMETER>', 'HUDSON_URL' = 'https://jenkins.edmcouncil.org/', 'ontology_publisher_current_product' = 'ontology', 'PROD_SPEC' = 'AboutIOFProd.rdf', 'WORKSPACE' = '/var/lib/jenkins/workspace/iof_master@2', '_' = '/usr/bin/java', 'GIT_COMMIT' = '8022447', 'ONTPUB_INPUT_REPOS' = 'iof', 'CI' = 'true', 'GIT_COMMENT' = 'merge Signed-off-by: mereolog <pawel.garbacz@makolab.com>', 'RJPP_BRANCH' = '*/develop', 'NGINX_SPEC_ROOT' = '/opt/purl.industrialontologies.org', 'JOB_BASE_NAME' = 'master', 'CP' = 'cp', 'RDFTOOLKIT_JAR' = '/usr/share/java/rdf-toolkit/rdf-toolkit.jar', 'verbose' = '1', 'HYGIENE_WARN_INCONSISTENCY_SPEC_FILE_NAME' = 'AboutIOFDev.rdf', 'BUILD_NUMBER' = '12', 'JIRA_ISSUE' = '', 'BUILD_URL' = 'https://jenkins.edmcouncil.org/job/iof/job/master/12/', 'RUN_ARTIFACTS_DISPLAY_URL' = 'https://jenkins.edmcouncil.org/job/iof/job/master/12/display/redirect?page=artifacts', 'GREP_OPTIONS' = '', 'INPUT' = '/input', 'ONTPUB_SPEC_HOST' = 'purl.industrialontologies.org', 'TAR' = 'tar', 'speedy' = '0', 'JOB_URL' = 'https://jenkins.edmcouncil.org/job/iof/job/master/', 'PS1' = '\h:\w\$ ', 'ONTOVIEWER_TOOLKIT_JAVA' = 'java -Dlogging.level.org.edmcouncil.spec.ontoviewer=INFO -Dlogging.level.org.edmcouncil.spec.ontoviewer.toolkit=DEBUG -Dlogging.level.org.springframework=INFO -jar /usr/share/java/onto-viewer/onto-viewer-toolkit.jar', 'spec_root' = '/output', 'LC_COLLATE' = 'C', 'FIND' = 'find', 'RUN_TESTS_DISPLAY_URL' = 'https://jenkins.edmcouncil.org/job/iof/job/master/12/display/redirect?page=tests', 'product_root_url' = 'https://purl.industrialontologies.org/iof/ontology', 'BUILD_TAG' = 'jenkins-iof-master-12', 'BASH_ENV' = '/etc/profile', 'OUTPUT' = '/output', 'JENA_BIN' = '/usr/share/java/jena/latest/bin', 'JENA_VERSION' = '3.17.0', 'TMPDIR' = '/var/tmp', 'product_branch_tag' = 'ontology/master/latest', 'JENA_HOME' = '/usr/share/java/jena/latest', 'JOB_DISPLAY_URL' = 'https://jenkins.edmcouncil.org/job/iof/job/master/display/redirect', 'GIT_BRANCH' = 'master', 'TREE' = 'tree', 'LS_COLORS' = 'rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=00:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.avif=01;35:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.webp=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:*~=00;90:*#=00;90:*.bak=00;90:*.old=00;90:*.orig=00;90:*.part=00;90:*.rej=00;90:*.swp=00;90:*.tmp=00;90:*.dpkg-dist=00;90:*.dpkg-old=00;90:*.ucf-dist=00;90:*.ucf-new=00;90:*.ucf-old=00;90:*.rpmnew=00;90:*.rpmorig=00;90:*.rpmsave=00;90:', 'tag_root_url' = 'https://purl.industrialontologies.org/iof/ontology/master/latest', 'branch_root' = '/output/iof/ontology/master', 'ONTPUB_IMAGE' = 'edmcouncil/ontology-publisher:stable', 'GIT_TAG_NAME' = 'latest', 'DEV_SPEC' = 'AboutIOFDev.rdf', 'BUILD_ID' = '12', 'ONTOVIEWER_TOOLKIT_JAR' = '/usr/share/java/onto-viewer/onto-viewer-toolkit.jar', 'LANG' = 'C.UTF-8', 'JOB_NAME' = 'iof/master', 'branch_tag' = 'master/latest', 'JENA3_JARS' = '.:/usr/share/java/jena/latest/lib/collection-0.7.jar:/usr/share/java/jena/latest/lib/commons-cli-1.4.jar:/usr/share/java/jena/latest/lib/commons-codec-1.15.jar:/usr/share/java/jena/latest/lib/commons-compress-1.20.jar:/usr/share/java/jena/latest/lib/commons-csv-1.8.jar:/usr/share/java/jena/latest/lib/commons-io-2.8.0.jar:/usr/share/java/jena/latest/lib/commons-lang3-3.11.jar:/usr/share/java/jena/latest/lib/httpclient-4.5.13.jar:/usr/share/java/jena/latest/lib/httpclient-cache-4.5.13.jar:/usr/share/java/jena/latest/lib/httpcore-4.4.13.jar:/usr/share/java/jena/latest/lib/jackson-annotations-2.11.3.jar:/usr/share/java/jena/latest/lib/jackson-core-2.11.3.jar:/usr/share/java/jena/latest/lib/jackson-databind-2.11.3.jar:/usr/share/java/jena/latest/lib/javax.annotation-api-1.3.2.jar:/usr/share/java/jena/latest/lib/jcl-over-slf4j-1.7.30.jar:/usr/share/java/jena/latest/lib/jena-arq-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-base-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-cmds-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-core-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-dboe-base-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-dboe-index-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-dboe-storage-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-dboe-trans-data-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-dboe-transaction-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-iri-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-rdfconnection-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-shacl-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-shaded-guava-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-tdb-3.17.0.jar:/usr/share/java/jena/latest/lib/jena-tdb2-3.17.0.jar:/usr/share/java/jena/latest/lib/jsonld-java-0.13.2.jar:/usr/share/java/jena/latest/lib/libthrift-0.13.0.jar:/usr/share/java/jena/latest/lib/log4j-api-2.14.0.jar:/usr/share/java/jena/latest/lib/log4j-core-2.14.0.jar:/usr/share/java/jena/latest/lib/log4j-slf4j-impl-2.14.0.jar:/usr/share/java/jena/latest/lib/slf4j-api-1.7.30.jar', 'source_family_root' = '/input/iof', 'NODE_LABELS' = 'docker idmp-docker', 'debug' = '1', 'EXECUTOR_NUMBER' = '3', 'ONTPUB_MERGED_INFIX' = '-Merged', 'HUDSON_SERVER_COOKIE' = 'e1c8575a462e2950', 'JENAROOT' = '/usr/share/java/jena/latest', 'spec_family_root_url' = 'https://purl.industrialontologies.org/iof', 'ONTPUB_IS_DARK_MODE' = '1', 'ROBOT_VERSION' = 'v1.9.0', 'CLASSPATH' = '', 'HYGIENE_ERROR_INCONSISTENCY_SPEC_FILE_NAME' = 'AboutIOFProd.rdf', 'branch_root_url' = 'https://purl.industrialontologies.org/iof/ontology/master', 'ONTPUB_FAMILY' = 'iof', 'RJPP_JENKINSFILE' = 'jenkins/iof/Jenkinsfile', 'HOSTNAME' = '7f1265271275', 'CHARSET' = 'UTF-8', 'GREP' = 'grep', 'BRANCH_NAME' = 'master', 'HOME' = '/', 2022-10-31 11:19:26.140 DEBUG 198 --- [ main] o.e.s.o.t.OntoViewerToolkitCommandLine : Property source 'Config resource 'class path resource [application.properties]' via location 'optional:classpath:/'' content: 'logging.level.org.springframework' = 'INFO', 'logging.level.org.semanticweb.owlapi' = 'WARN', 'logging.level.org.edmcouncil.spec.ontoviewer' = 'INFO', 'ov.application.name' = 'OntoViewer Toolkit', 'ov.application.version' = '0.4.0', 'app.defaultHomePath' = '*', 2022-10-31 11:19:26.140 DEBUG 198 --- [ main] o.e.s.o.t.OntoViewerToolkitCommandLine : Raw command line arguments: [--data, /input/iof/AboutIOFProd.rdf, --output, /var/tmp/console.txt, --ontology-mapping, /input/iof/catalog-v001.xml, --goal, consistency-check] 2022-10-31 11:19:26.259 WARN 198 --- [ main] o.e.s.o.t.OntoViewerToolkitCommandLine : Error while handling ontology mapping from path '/input/iof/catalog-v001.xml'. Details: Unable to load catalog with ontology mappings. org.edmcouncil.spec.ontoviewer.core.exception.OntoViewerException: Unable to load catalog with ontology mappings. at org.edmcouncil.spec.ontoviewer.core.mapping.OntologyCatalogParser.readOntologyMapping(OntologyCatalogParser.java:22) ~[onto-viewer-core-0.4.0.jar!/:0.4.0] at org.edmcouncil.spec.ontoviewer.toolkit.OntoViewerToolkitCommandLine.populateConfiguration(OntoViewerToolkitCommandLine.java:186) ~[classes!/:0.4.0] at org.edmcouncil.spec.ontoviewer.toolkit.OntoViewerToolkitCommandLine.run(OntoViewerToolkitCommandLine.java:106) ~[classes!/:0.4.0] at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:791) ~[spring-boot-2.5.2.jar!/:2.5.2] at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:775) ~[spring-boot-2.5.2.jar!/:2.5.2] at org.springframework.boot.SpringApplication.run(SpringApplication.java:345) ~[spring-boot-2.5.2.jar!/:2.5.2] at org.springframework.boot.SpringApplication.run(SpringApplication.java:1343) ~[spring-boot-2.5.2.jar!/:2.5.2] at org.springframework.boot.SpringApplication.run(SpringApplication.java:1332) ~[spring-boot-2.5.2.jar!/:2.5.2] at org.edmcouncil.spec.ontoviewer.toolkit.OntoViewerToolkitApplication.main(OntoViewerToolkitApplication.java:10) ~[classes!/:0.4.0] at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na] at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na] at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na] at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na] at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49) ~[onto-viewer-toolkit.jar:0.4.0] at org.springframework.boot.loader.Launcher.launch(Launcher.java:108) ~[onto-viewer-toolkit.jar:0.4.0] at org.springframework.boot.loader.Launcher.launch(Launcher.java:58) ~[onto-viewer-toolkit.jar:0.4.0] at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:88) ~[onto-viewer-toolkit.jar:0.4.0] Caused by: org.simpleframework.xml.core.ElementException: Element 'group' does not have a match in class org.edmcouncil.spec.ontoviewer.core.mapping.model.Catalog at line 11 at org.simpleframework.xml.core.Composite.readElement(Composite.java:527) ~[simple-xml-2.7.1.jar!/:na] at org.simpleframework.xml.core.Composite.readElements(Composite.java:445) ~[simple-xml-2.7.1.jar!/:na] at org.simpleframework.xml.core.Composite.access$400(Composite.java:59) ~[simple-xml-2.7.1.jar!/:na] at org.simpleframework.xml.core.Composite$Builder.read(Composite.java:1383) ~[simple-xml-2.7.1.jar!/:na] at org.simpleframework.xml.core.Composite.read(Composite.java:201) ~[simple-xml-2.7.1.jar!/:na] at org.simpleframework.xml.core.Composite.read(Composite.java:148) ~[simple-xml-2.7.1.jar!/:na] at org.simpleframework.xml.core.Traverser.read(Traverser.java:92) ~[simple-xml-2.7.1.jar!/:na] at org.simpleframework.xml.core.Persister.read(Persister.java:625) ~[simple-xml-2.7.1.jar!/:na] at org.simpleframework.xml.core.Persister.read(Persister.java:606) ~[simple-xml-2.7.1.jar!/:na] at org.simpleframework.xml.core.Persister.read(Persister.java:584) ~[simple-xml-2.7.1.jar!/:na] at org.simpleframework.xml.core.Persister.read(Persister.java:562) ~[simple-xml-2.7.1.jar!/:na] at org.simpleframework.xml.core.Persister.read(Persister.java:462) ~[simple-xml-2.7.1.jar!/:na] at org.edmcouncil.spec.ontoviewer.core.mapping.OntologyCatalogParser.readOntologyMapping(OntologyCatalogParser.java:20) ~[onto-viewer-core-0.4.0.jar!/:0.4.0] ... 16 common frames omitted
```

This may have been cause by the following catalog-v001.xml:

``` 
 <?xml version="1.0" encoding="UTF-8" standalone="no"?> <catalog prefer="public" xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog"> <uriid="User Entered Import Resolution"name="https://www.industrialontologies.org/ontology/supplychain/SupplyChainReferenceOntology/"uri="./supplychain/SupplyChainReferenceOntology.rdf"/> <uriid="User Entered Import Resolution"name="https://purl.industrialontologies.org/ontology/maintenance/Maintenance/"uri="./maintenance/Maintenance.rdf"/> <uriid="User Entered Import Resolution"name="https://purl.industrialontologies.org/ontology/core/Core/"uri="./core/Core.rdf"/> <uriid="User Entered Import Resolution"name="https://purl.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/"uri="./core/meta/AnnotationVocabulary.rdf"/> <uriid="User Entered Import Resolution"name="https://purl.industrialontologies.org/ontology/core/20220506/Core/"uri="./core/Core.rdf"/> <groupid="Folder Repository, directory=, recursive=true, Auto-Update=true, version=2"prefer="public"xml:base=""/> </catalog>  
```

--- a/onto-viewer-core/src/main/java/org/edmcouncil/spec/ontoviewer/core/mapping/model/Catalog.java
+++ b/onto-viewer-core/src/main/java/org/edmcouncil/spec/ontoviewer/core/mapping/model/Catalog.java
@@ -5,7 +5,7 @@ import org.simpleframework.xml.Attribute;
 import org.simpleframework.xml.ElementList;
 import org.simpleframework.xml.Root;
 
-@Root
+@Root(strict = false)
 public class Catalog {
 
   @Attribute

